src/runtime_src/ert/scheduler/scheduler.cpp

```diff
@@ -125,39 +125,38 @@ static const u32 AP_CTRL_ME    = 3;
 ////////////////////////////////////////////////////////////////
 // Extensions to core/include/ert.h
 ////////////////////////////////////////////////////////////////
-const addr_type STATUS_REGISTER_ADDR[4] =
+addr_type STATUS_REGISTER_ADDR[4] =
 {
   ERT_STATUS_REGISTER_ADDR0
  ,ERT_STATUS_REGISTER_ADDR1
  ,ERT_STATUS_REGISTER_ADDR2
  ,ERT_STATUS_REGISTER_ADDR3
 };
 
-const addr_type CU_DMA_REGISTER_ADDR[4] =
+addr_type CU_DMA_REGISTER_ADDR[4] =
 {
   ERT_CU_DMA_REGISTER_ADDR0
  ,ERT_CU_DMA_REGISTER_ADDR1
  ,ERT_CU_DMA_REGISTER_ADDR2
  ,ERT_CU_DMA_REGISTER_ADDR3
 };
 
-const addr_type CU_STATUS_REGISTER_ADDR[4] =
+addr_type CU_STATUS_REGISTER_ADDR[4] =
 {
   ERT_CU_STATUS_REGISTER_ADDR0
  ,ERT_CU_STATUS_REGISTER_ADDR1
  ,ERT_CU_STATUS_REGISTER_ADDR2
  ,ERT_CU_STATUS_REGISTER_ADDR3
 };
 
-const addr_type CQ_STATUS_REGISTER_ADDR[4] =
+addr_type CQ_STATUS_REGISTER_ADDR[4] =
 {
   ERT_CQ_STATUS_REGISTER_ADDR0
  ,ERT_CQ_STATUS_REGISTER_ADDR1
  ,ERT_CQ_STATUS_REGISTER_ADDR2
  ,ERT_CQ_STATUS_REGISTER_ADDR3
 };
```

Comment:
Maybe remove this up front initialization and just declare / define like this:
```
addr_type STATUS_REGISTER_ADDR[4] = {0,0,0,0};
...
addr_type CQ_STATUS_REGISTER[4] = {0,0,0,0};
```
Then unconditionally initialize the values for all platforms in setup_ert_base_addr() (see  comment below) after conditionally reading the ert_base_addr for ERT_BUILD_V20 also in setup_ert_base_addr().

Reply:
done.